### ISSUE TITLE
[gatsby-plugin-preact] fast-refresh/prefresh stops working on ^4.0.11

### ISSUE THREAD

**Issue by chrsep** [gatsbyjs/gatsby#27151]
## Description

Hi, it seems that fast-refresh/prefresh stops working while using gatsby-plugin-preact version ^4.0.11. It works if I revert back to 4.0.10. Seems like the only change on 4.0.11 is an upgrade of  @prefresh/webpack to version ^1.0.2 from ^0.8.1. 

### Steps to reproduce

Can be reproduced on a barebone gatsby + preact project that i've made.
1. Clone https://github.com/chrsep/test-prefresh
2.  yarn install && yarn develop
3. Open localhost:8000 in the browser and change something on src/pages/index.js

### Expected result

Changes should be reflected directly on the browser without reloading the page after saving the file.

### Actual result

Nothing happens on the browser, changes only get applied after refreshing the page.

### Environment
```
  System:
    OS: Linux 5.8 Arch Linux
    CPU: (8) x64 Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
    Shell: 3.1.2 - /usr/bin/fish
  Binaries:
    Node: 13.13.0 - ~/.config/nvm/13.13.0/bin/node
    Yarn: 1.22.5 - /usr/bin/yarn
    npm: 6.14.4 - ~/.config/nvm/13.13.0/bin/npm
  Languages:
    Python: 3.8.5 - /usr/bin/python
  Browsers:
    Firefox: 81.0
  npmPackages:
    gatsby: ^2.24.63 => 2.24.63 
    gatsby-image: ^2.4.19 => 2.4.19 
    gatsby-plugin-manifest: ^2.4.30 => 2.4.30 
    gatsby-plugin-offline: ^3.2.28 => 3.2.28 
    gatsby-plugin-preact: 4.0.12 => 4.0.12 
    gatsby-plugin-react-helmet: ^3.3.11 => 3.3.11 
    gatsby-plugin-sharp: ^2.6.36 => 2.6.36 
    gatsby-source-filesystem: ^2.3.30 => 2.3.30 
    gatsby-transformer-sharp: ^2.5.15 => 2.5.15 

```

**Comment by wardpeet** [comment]
I'll publish a new version today and test it with your site first. :) Thanks for reporting

**Comment by wardpeet** [comment]
Published in gatsby-plugin-preact@4.0.15
